src/cli/cli.cpp

```diff
@@ -2742,6 +2766,19 @@ template <> otError Interpreter::Process<Cmd("log")>(Arg aArgs[])
         }
     }
 #if (OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_DEBUG_UART) && OPENTHREAD_POSIX
+    /**
+     * @cli log filename
+     * @par
+     * Specifies filename to capture otPlatLog() messages, useful when debugging
+     * automated test scripts on Linux when logging disrupts the automated test scripts.
+     * @par
+     * Note: Simulation only and `OPENTHREAD_EXAMPLES_SIMULATION` is true.
```

Comment:
@abtink is this statement still true (taken from the CLI README)?  OPENTHREAD_EXAMPLES_SIMULATION doesn't need to be set for this command to be available.

Reply:
Sorry! I just saw this.
I am not sure. Need to check it in code.

Reply:
Yes. looks like this is limited to simulation platform (which uses the `OPENTHREAD_CONFIG_LOG_OUTPUT_DEBUG_UART` as its log-output).